app/controllers/hyrax/admin/collection_type_participants_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hyrax v3.4.2 Add backend validation to stop admin group access from being destroyed
+require_dependency(
+  Hyrax::Engine
+    .root
+    .join('app', 'controllers', 'hyrax', 'admin', 'collection_type_participants_controller')
+    .to_s
+)
+
+Hyrax::Admin::CollectionTypeParticipantsController.class_eval do
```

Comment:
I'm fairly sure this will cause "double load" errors in development. is there a strong reason not to use a decorator or complete override for this?